steering_vectors/train_steering_vector.py

```diff
@@ -16,18 +16,22 @@
 class SteeringVectorTrainingSample(NamedTuple):
```

Comment:
nit: we should change this to a dataclass at this point since these optional params don't make a lot of sense as tuples IMO

Reply:
Resolved by [be9fb91](https://github.com/steering-vectors/steering-vectors/pull/21/commits/be9fb919720b59a0d099cf0f1b4a6de0c6676ba7)